### ISSUE TITLE
Classification tutorial: (224,224,1) doesn't match the broadcast shape (224,224,3)

### ISSUE THREAD

**Issue by dlazerka** [catalyst-team/catalyst#513]
**To Reproduce**
1. Run on Google Collab (no single change made)
https://colab.research.google.com/github/catalyst-team/catalyst/blob/master/examples/notebooks/classification-tutorial.ipynb

Observed:
Cell 31 (`runner.train()`):
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-32-bb8ae99f70ba> in <module>()
     21     ],
     22     num_epochs=NUM_EPOCHS,
---> 23     verbose=True
     24 )

10 frames
/usr/local/lib/python3.6/dist-packages/torch/_utils.py in reraise(self)
    383             # (https://bugs.python.org/issue2651), so we work around it.
    384             msg = KeyErrorMessage(msg)
--> 385         raise self.exc_type(msg)

ValueError: Caught ValueError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/torch/utils/data/_utils/worker.py", line 178, in _worker_loop
    data = fetcher.fetch(index)
  File "/usr/local/lib/python3.6/dist-packages/torch/utils/data/_utils/fetch.py", line 44, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/usr/local/lib/python3.6/dist-packages/torch/utils/data/_utils/fetch.py", line 44, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/usr/local/lib/python3.6/dist-packages/catalyst/data/dataset.py", line 78, in __getitem__
    dict_ = self.dict_transform(dict_)
  File "/usr/local/lib/python3.6/dist-packages/torchvision/transforms/transforms.py", line 70, in __call__
    img = t(img)
  File "/usr/local/lib/python3.6/dist-packages/catalyst/data/augmentor.py", line 23, in __call__
    ] = self.augment_fn(dict_[self.dict_key], **self.default_kwargs)
  File "<ipython-input-22-75f5af7a01a3>", line 14, in <lambda>
    augment_fn=lambda x: train_transforms(image=x)["image"]
  File "/usr/local/lib/python3.6/dist-packages/albumentations/core/composition.py", line 171, in __call__
    data = t(**data)
  File "/usr/local/lib/python3.6/dist-packages/albumentations/core/transforms_interface.py", line 38, in __call__
    res[key] = target_function(arg, **dict(params, **target_dependencies))
  File "/usr/local/lib/python3.6/dist-packages/albumentations/augmentations/transforms.py", line 808, in apply
    return F.normalize(image, self.mean, self.std, self.max_pixel_value)
  File "/usr/local/lib/python3.6/dist-packages/albumentations/augmentations/functional.py", line 93, in normalize
    img -= mean
ValueError: non-broadcastable output operand with shape (224,224,1) doesn't match the broadcast shape (224,224,3)
```

**Comment by TezRomacH** [comment]
Hi! Thanks for the bug report, I'll be sure to check it out.

**Comment by Scitator** [comment]
resolved by 19.11.4 version (there was a bug in 19.11.3)
